chapters/appendix-IX-file-tags.md

```diff
@@ -0,0 +1,45 @@
+# Appendix IX: SPDX File Tags
+
+## Rationale
+
+SPDX short-form license identifiers, described in [Appendix V](./appendix-V-using-SPDX-short-identifiers-in-source-files.md), provide a mechanism for developers to easily convey information about the licenses they declare on a file-by-file basis. That mechanism is intentionally very easy for software tools to identify and detect, since it includes a standard text string (`SPDX-License-Identifier:`) that is unlikely to occur in other contexts, and since it uses license identifiers from the [SPDX License List](https://spdx.org/licenses) or from user-defined `LicenseRef-` statements.
+
+The SPDX specification defines various other fields in the [File Information](./4-file-information.md) section that are also useful for conveying information on a file-by-file basis. This appendix describes a mechanism, similar to `SPDX-License-Identifier`, for developers to convey such other file-based information easily in comments in their files. This in turn enables software tools to easily find and extract that information, and to insert it into the corresponding fields of an SPDX document generated by those tools.
+
+## Format
+
+An SPDX file tag consists of a single line, generally as part of a comment, in the following format:
```

Comment:
I would suggest to specify "as part of a comment near the top of the file". The current wording suggests that any comment will do, which it well may, but the header comment is probably preferable.

Reply:
Great suggestion -- now updated accordingly  :)  Thank you @carmenbianca!